javascript/exorelay/src/websocket-connector/websocket-connector.ls

```diff
@@ -27,6 +27,7 @@ class WebSocketConnector extends EventEmitter
   close: ->
     return unless @socket
     debug "no longer connected at 'ws://#{@exocom-host}/#{@exocom-port}'"
+    @socket-killed = yes
```

Comment:
Should we reset `socket-killed` to no somewhere? Otherwise after closing it, if you were to connect again it would not have the reconnect behavior.